### PR TITLE
fixed path to pictures, created openProfilePage pre-condition; moved one test case (TUA 359)

### PR DESCRIPTION
**Pull request by janestadnytska**

### CODE REVIEW

File: src/test/java/org/ssu/edu/teachua/ui/TUA231.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public void testTUA231() throws IOException {
                 .fillSortNumber(VALID_SORT_NUMBER)
                 .fillTitle(VALID_TITLE)
                 .fillDescription(VALID_DESCRIPTION)
-                .addPhoto(valueProvider.getFilePath("photos/image.png"));
+                .addPhoto(valueProvider.getFilePath("image.png"));
```
Comment (lhalam) [inline src/test/java/org/ssu/edu/teachua/ui/TUA231.java:53]: ???

File: src/test/java/org/ssu/edu/teachua/utils/TestValueProvider.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public String getBaseUiUrl() {
 
     public String getFilePath(String fileName) {
         return Paths.get(Paths.get(System.getProperty("user.dir")).toString(),
-                "src", "test", "resources", fileName).toString();
+                "src", "test", "resources", "photos", fileName).toString();
```
Comment (lhalam) [inline src/test/java/org/ssu/edu/teachua/utils/TestValueProvider.java:48]: ???

**Review by lhalam — COMMENTED** [review]
if I need a path other than pictures?

File: src/test/java/org/ssu/edu/teachua/ui/TUA231.java (hunk 1 above)
Comment (janestadnytska) [reply src/test/java/org/ssu/edu/teachua/ui/TUA231.java:53]: image
we added directory for images

Comment (janestadnytska) [reply src/test/java/org/ssu/edu/teachua/ui/TUA231.java:53]: image
and I fixed that here

File: src/test/java/org/ssu/edu/teachua/utils/TestValueProvider.java (hunk 2 above)
Comment (janestadnytska) [reply src/test/java/org/ssu/edu/teachua/utils/TestValueProvider.java]: I reverted the changes back

**Review by lhalam — APPROVED** [review]

File: src/test/java/org/ssu/edu/teachua/ui/TUA231.java (hunk 1 above)
Comment (janestadnytska) [reply src/test/java/org/ssu/edu/teachua/ui/TUA231.java]: I reverted the changes back

**Pull request merged**